firebase-firestore/src/main/java/com/google/firebase/firestore/remote/RemoteStore.java

```diff
@@ -407,8 +416,20 @@ private void startWatchStream() {
         "startWatchStream() called when shouldStartWatchStream() is false.");
     watchChangeAggregator = new WatchChangeAggregator(this);
     watchStream.start();
-
     onlineStateTracker.handleWatchStreamStart();
+
+    DelayedTask connectivityAttemptTimer =
+        workerQueue.enqueueAfterDelay(
+            TimerId.ONLINE_STATE_TIMEOUT,
```

Comment:
If this change goes trough, `TimerId.ONLINE_STATE_TIMEOUT` should be changed to `TimerId.CONNECTVITY_ATTEMPT_TIMEOUT`, and ported to web/iOS. I can't change that right now since the spec tests refer to the timerId as `ONLINE_STATE_TIMEOUT`

Reply:
This can be handled by adding an additional string literal check in the spec test. Optional as to whether or not that's easier to deal with than this leaving this alone for now.